src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberFieldSymbol.cs

```diff
@@ -323,7 +323,7 @@ public sealed override bool HasInitializer
             get { return _hasInitializer; }
         }
 
-        public VariableDeclaratorSyntax VariableDeclaratorNode
+        private VariableDeclaratorSyntax VariableDeclaratorNode
```

Comment:
Thank you for making the change.

---
In reply to: [80965746](https://github.com/dotnet/roslyn/pull/13999#discussion_r80965746) [](ancestors = 80965746,80959907,80850539,80816186,80815616)